index.js

```diff
@@ -0,0 +1,128 @@
+
+const tripAPI = "https://trektravel.herokuapp.com/trips";
+
+const reportStatus = (message) => {
+    $('#status-message').html(message);
+};
+
+const reportError = (message, errors) => {
+    let content = `<p>${message}</p><ul>`;
+    for (const field in errors) {
+        for (const problem of errors[field]) {
+            content += `<li>${field}: ${problem}</li>`;
+        }
+    }
+    content += "</ul>";
+    reportStatus(content);
+};
+
+const displayTripList = () => {
+    const target = $('#trip-list');
+    target.empty();
+    axios.get(tripAPI)
+        .then((response) => {
+            response.data.forEach((trip) => {
+                target.append(`<button id="trip-${trip.id}">${trip.name}</button>`);
+                $(`#trip-${trip.id}`).click(showTripDetails(trip.id));
+            });
+        })
+        .catch((error) => {
+            if (error.response.data && error.response.data.errors) {
```

Comment:
It would also be appropriate to have `error.response` in your if statement to verify if there is a response object in error.  Network errors won't have this.